src/java.desktop/macosx/native/libjsound/PLATFORM_API_MacOSX_Ports.cpp

```diff
@@ -229,6 +237,9 @@ INT32 PORT_GetPortMixerDescription(INT32 mixerIndex, PortMixerDescription* mixer
 void* PORT_Open(INT32 mixerIndex) {
     TRACE1("\n>>PORT_Open (mixerIndex=%d)\n", (int)mixerIndex);
     PortMixer *mixer = (PortMixer *)calloc(1, sizeof(PortMixer));
+    if (!mixer) {
+        return nullptr;
+    }
```

Comment:
Pre-existing bug: the caller code is ready for null: https://github.com/openjdk/jdk/blob/62d165d0e4e5ab1bcef35d5031554a354052c6d5/src/java.desktop/share/classes/com/sun/media/sound/PortMixer.java#L76 -> https://github.com/openjdk/jdk/blob/62d165d0e4e5ab1bcef35d5031554a354052c6d5/src/java.desktop/share/native/libjsound/PortMixer.c#L45 -> PORT_Open